Assets/_Project/Scripts/UI/Bilboard.cs

```diff
@@ -0,0 +1,18 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+public class Bilboard : MonoBehaviour
```

Comment:
This should be `Billboard`, but again could use a much more descriptive name.  I had to go find the actual component in the editor and remove it to find out what it actually does.  I would suggest something like `Nameplate` or `WorldUIElement`.  Something more descriptive would be much better